src/core/public/legacy/legacy_service.ts

```diff
@@ -55,37 +55,12 @@ export class LegacyPlatformService {
   constructor(private readonly params: LegacyPlatformParams) {}
 
   public setup({ core, plugins }: SetupDeps) {
-    const {
-      application,
-      i18n,
-      injectedMetadata,
-      fatalErrors,
-      notifications,
-      http,
-      uiSettings,
-      chrome,
-    } = core;
     // Inject parts of the new platform into parts of the legacy platform
     // so that legacy APIs/modules can mimic their new platform counterparts
     require('ui/new_platform').__newPlatformSetup__(core, plugins);
-    require('ui/metadata').__newPlatformSetup__(injectedMetadata.getLegacyMetadata());
```

Comment:
:+1: very nice removing all these integration points, just `ui/new_platform` is enough